From 1d26e4d72f2d0563cc6455e682a5d4c491de178c Mon Sep 17 00:00:00 2001
From: Ben Wolsieffer <benwolsieffer@gmail.com>
Date: Sun, 9 Dec 2018 16:46:00 -0500
Subject: [PATCH] dtoverlays: fe-pi-audio: fix sgtl5000 compatible
 string

The compatible string was set to "fepi,sgtl5000", which worked for some
reason in 4.14, but does not work in 4.19, presumably due to some
change in the kernel matching logic. The correct string is
"fsl,sgtl5000".

Signed-off-by: Ben Wolsieffer <benwolsieffer@gmail.com>
---
 arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
@@ -39,7 +39,7 @@
 
 			sgtl5000@0a {
 				#sound-dai-cells = <0>;
-				compatible = "fepi,sgtl5000";
+				compatible = "fsl,sgtl5000";
 				reg = <0x0a>;
 				clocks = <&sgtl5000_mclk>;
 				micbias-resistor-k-ohms = <2>;
